scripting/pugsetup.sp

```diff
@@ -2237,3 +2243,16 @@ stock bool PermissionFromString(const char[] permissionString, Permission& p,
 
   return true;
 }
+
+public void OnClientDisconnect(int client)
+{
+  if(GetClientCount() <= 2)
```

Comment:
It's a little confusing that the cvar description refers to "1 player", but this check is against 2 or less players.

I would suggest performing the check in the `OnClientDisconnect_Post` forward instead, and then the check can compare against 1 instead of 2.

Reply:
Oops.
Fixed.